Common/OpenCL/ITKimprovements/itkOpenCLContext.h

```diff
@@ -469,7 +469,7 @@ class ITKOpenCL_EXPORT OpenCLContext : public LightObject
 
   ITK_OPENCL_DECLARE_PRIVATE( OpenCLContext )
 
-  std::auto_ptr< OpenCLContextPimpl > d_ptr;
+  std::unique_ptr< OpenCLContextPimpl > d_ptr;
```

Comment:
👍  to "ENH: Use unique_ptr instead of auto_ptr". As long as it compiles, it's fine, I guess.